tests/core/nlg/test_response.py

```diff
@@ -80,6 +80,27 @@ async def test_nlg_when_multiple_conditions_satisfied():
     assert resp.get("text") in ["example A", "example B"]
 
 
+async def test_nlg_slot_case_sensitivity():
+    responses = {
+        "utter_action": [
+            {
+                "text": "example a",
```

Comment:
I'd extract this in a variable `response_text`, this makes the assertion cleaner in line 101.